randobot/zsr.py

```diff
@@ -123,8 +124,9 @@ def roll_seed(self, preset, encrypt, dev, settings):
                 req_body = json.dumps(self.presets[preset]['settings'])
             # Fetch tournament preset and patch with drafted settings.
             else:
-                self.presets_dev.get('s7').get('settings').update(settings)
-                req_body = json.dumps(self.presets_dev.get('s7').get('settings'))
+                self.handle_conditionals(dev, settings)
+                self.presets.get('s7').get('settings').update(settings)
```

Comment:
I can't remember if I mentioned this in the last review or not, but this `update` will affect the preset default values which will likely cause some seed bleed to happen. The `settings` dict is safely mutable though, so we can avoid that problem by just reversing this logic:: `settings.update(self.presets.get('s7').get('settings'))`

Reply:
You definitely did and I just forgot about it. I think if I did it that way it would update the drafted values with the values from the base preset? I decided to just move the entire function into the RandoHandler class to hopefully avoid that issue. Patching is done in there and then passed to the roll function later on